src/auth.js

```diff
@@ -0,0 +1,51 @@
+import jsonwebtoken from "jsonwebtoken";
+
+const USERNAME = "user";
+const PASSWORD = "password";
+const tokens = [];
+const JWT_SECRET = "blahblahblahblahBLAHBLAHBLAHBLAHBLAHBLAHBLAHBLAHBLAAAA";
+const FIVE_MINS = 5 * 60 * 1000;
+
+export const login = (req, res) => {
+	const authHeader = req.headers.authentication;
+  const b64creds = authHeader.slice(6);
+  const bufferedInput = Buffer.from(b64creds, "base64")
+  const creds = bufferedInput.toString();
+	const [user, password] = creds.split(":");
+  
+  if (user == USERNAME && password == PASSWORD) {
+		const jwt = jsonwebtoken.sign(
+			{
+				username: user
+			},
+			JWT_SECRET,
+			{ expiresIn: FIVE_MINS } 
+		);
+		tokens.push(jwt);
+		return res.status(200).json({
+			token: jwt
+		});
+	} else {
+		return res
```

Comment:
Prettier doesn't always change line breaks. If you try and type `return res.status(401).json({ message: "The username and password you provided are invalid" });` all on one line, it will insert line breaks at the dots. If you instead do a line break before 'message' it will let you keep it the way it is. 🤷🏻‍♀️